examples/plain-js/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@examples/plain-js",
   "version": "0.25.1-canary.1",
   "scripts": {
+    "dev": "node server",
```

Comment:
will make a separate example for custom server, just keeping it here whilst developing